src/main/java/com/information/country/ProvinceAndState.java

```diff
@@ -0,0 +1,19 @@
+package com.information.country;
+
+public class ProvinceAndState {
+    private final String provinceAndStateName;
```

Comment:
ProvinceAndStateクラスなので、nameだけでもOKだと思います。
```suggestion
    private final String name;
```

Reply:
既にProvinceAndStateクラスを作っているので,その中の場合はnameだけでも良いのですね。
承知いたしました！